### PR TITLE
switch router->ragent traffic over to TLS

### PR DESCRIPTION
**Pull request by grs**

### CODE REVIEW

File: templates/install/openshift/enmasse-with-sasldb.yaml [hunk 1]
```diff
@@ -40,6 +40,8 @@ objects:
               value: "${ADDRESS}"
             - name: CLUSTER_ID
               value: "${CLUSTER_ID}"
+            - name: PN_TRACE_FRM
+              value: '1'
```
Comment (lulf) [inline templates/install/openshift/enmasse-with-sasldb.yaml]: Perhaps we should remove this file, it is not used (or usable for that sake) anymore.

File: ragent/tls_options.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2016 Red Hat Inc.
```
Comment (lulf) [inline ragent/tls_options.js]: Update year.

File: ragent/tls_options.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+var path = require('path');
+var fs = require('fs');
+
+function get_client_options(config) {
+    var options = config || {};
+    var cert_dir = (process.env.CERT_DIR !== undefined) ? process.env.CERT_DIR : '/etc/enmasse-certs';
+    var ca_path = path.resolve(cert_dir, 'ca.crt');//TODO: allow setting via env var
+    var crt_path = path.resolve(cert_dir, 'tls.crt');//TODO: allow setting via env var
+    var key_path = path.resolve(cert_dir, 'tls.key');//TODO: allow setting via env var
```
Comment (lulf) [inline ragent/tls_options.js]: IMHO setting the CERT DIR is sufficient, and in doing testing so far I've not had any use for setting them individually.

I'd say just fix them or remove them?

File: ragent/tls_options.js (hunk 2 above)
Comment (grs) [reply ragent/tls_options.js]: done

File: ragent/tls_options.js (hunk 3 above)
Comment (grs) [reply ragent/tls_options.js]: done

File: templates/install/openshift/enmasse-with-sasldb.yaml (hunk 1 above)
Comment (grs) [reply templates/install/openshift/enmasse-with-sasldb.yaml]: done

**Pull request merged**